Gaijinizer patch/Map070.txt

```diff
@@ -2119,29 +2187,31 @@ spirit of Haru.#MTLed
 
 そう不安になりつつも、彼女に任せてみたんだ。
 > CONTEXT: Map070/events/9/pages/1/248/Dialogue/enemy7
-"This is because I'm only living at home
-You don't know human fear? ". 
-Even though I became uneasy, I tried to leave it
-to her.#MTLed
+I thought that the only reason she could propose
+such a thing was because as a housecat, she had
+no concept of how awful humans could be. But
+despite my reservations, I went along with it.#Ron
 > END STRING
 
 > BEGIN STRING
 でも、知らなかったのは私の方だった。
 
  人間が怒声じゃなく、あんな情けない声も出すなんて。
 > CONTEXT: Map070/events/9/pages/1/253/Dialogue/enemy7
-But it was me who did not know.
-Human beings are not angry voices, they also make
-such unshy voices.#MTLed
+But it turned out I was the one who had no real
+concept of human nature. Humans were capable of
+more than angry voices, they could make such
+unseemly voices too.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 ハルの言いなりになって、夢中で甘える人間の姿は
 
 私にとって、驚かすよりも良い気味だった。
 > CONTEXT: Map070/events/9/pages/1/257/Dialogue/enemy7
-Haru's obsessive, Indulge Yourself human figure
-It was better for me than I was surprised.#MTLed
+The sight of them falling for Haru, becoming her
+obsessive yes-men, it felt good to me rather than
+surprising.#RonIffyWording
```

Comment:
This is a bit of a rewording, but I think it flows better. Feel free to disagree with some of this if you want.
>_**The sight of them falling for Haru, becoming
lovers obsessed with her...
It surprised me a lot. It made me feel so good.**_